Makefile

```diff
@@ -74,7 +76,7 @@ check:
 # To avoid unintended conflicts with file names, always add to .PHONY
 # unless there is a reason not to.
 # https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
-.PHONY: build build_gui
+.PHONY: build build_gui build_daemon build_wallet build_shell
```

Comment:
Please revert this line.

Reply:
https://github.com/pactus-project/pactus/pull/1183/commits/882407bf5b7ad0673475acb8ee1e634640a252d4